lib/types/array.js

```diff
@@ -342,7 +342,13 @@ module.exports = Any.extend({
 
                 if (ordereds.length) {
                     internals.fillOrderedErrors(schema, errors, ordereds, value, state, prefs);
+
+                    if (errors.length) {
```

Comment:
Looks okay to me 👍 However, I've got mixed feelings about this nested if statement, but okay does the job.

Reply:
I think there's no other option, nested `if` are used a lot in the same file, doesn't look odd.